examples/bri-3/src/bri/zeroKnowledgeProof/agents/ccsmAnchorStorage.agent.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@nestjs/common';
+import { BlockchainService } from '../components/blockchain/blockchain.service';
+
+import { CCSMAnchor } from '../models/ccsmAnchor';
+import { ZeroKnowledgeProofVerificationInput } from '../models/zeroKnowledgeProofVerificationInput';
+
+@Injectable()
+export class CCSMAnchorStorageAgent {
+  constructor(private readonly storageAgent: BlockchainService) {}
```

Comment:
CCSMAnchorStorageAgent  should inject the blokcchainService and call store on it.

CCSMAnchorStorageAgent is a storage agent which is injecting a service that helps it to talk to the storage layer - in this case blockchain.